### ISSUE TITLE
Deye SUN 12K-SG04LP3-EU

### ISSUE THREAD

**Issue by Voelk-IT** [s10l/dbus-deye-sun-g3-pvinverter#5]
Hello, I have a question.

Would it be possible for you to publish a version of how I could use the Deye SUN 12K-SG04LP3-EU with it?

Would be really a mega thing

**Comment by s10l** [comment]
Yes, it is possible. However, I don't have a unit to test. Would you be willing to test?
I assume that we need a few iterations.

**Comment by janekg99** [comment]
Hello,
is there any progress or update regarding 3 phase? Interested for SUN-20K-SG05LP3 together with Cerbo GX.
Also question - does it work with Venus OS installed on Raspberry Pi?

Kind regards
Janek
